src/baguetteBox.js

```diff
@@ -603,6 +603,15 @@
         }
         return returnValue;
     }
+    
+    // opens specific image
+    function showImage(selector, imageIndex) {
+	    if ( null === imageIndex || undefined === imageIndex || '' === imageIndex ){
+		    imageIndex = 0;
+	    }
+	    var activeImage = baguetteBox.data[selector].galleries[0][imageIndex];
+	    activeImage.eventHandler(activeImage);
```

Comment:
We need to check that given index doesn't exceed the elements count:
```js
if (activeImage) {
    activeImage.eventHandler(new MouseEvent('click'))
}
```
I've also replaced `activeImage` argument, because the method expects an event object.